### DISCUSSION TITLE
Monterey

### DISCUSSION THREAD

**Discussion by mister5111** [strongswan/strongswan#751] [Q&A]
I have an application that runs onipsec ikev2
This application uses the strongswan module with org.strongswan.charon-xpc
After upgrading to monterey I get an error like #707
How do I build the new org.strongswan.charon-xpc module?

**Comment by tobiasbrunner** [comment]
Sorry, can't help you with that (refer to the previous discussions).

**Reply by mister5111** [reply]
How can I build the new org.strongswan.charon-xpc module?
